subprojects/veins_inet/src/veins_inet/VeinsInetManager.h

```diff
@@ -37,13 +38,17 @@ namespace veins {
  * @author Christoph Sommer
  *
  */
-class VEINS_INET_API VeinsInetManager : public TraCIScenarioManagerLaunchd {
+class VEINS_INET_API VeinsInetManager : public TraCIScenarioManagerLaunchd, public cListener {
```

Comment:
is this modification (still) neeeded?

Reply:
Good catch. No, it's leftover from implementing the signal handling without using the signal manager and can be removed.